src/app/board/task/task.component.html

```diff
@@ -0,0 +1,8 @@
+<section class="tasks-wrapper">
+  <div class="task-wrapper">
```

Comment:
do you need to use this class for both tags?

Reply:
these are different classes)